common-lb/src/test/java/loadbalancer/LoadBalancerServerTest.java

```diff
@@ -0,0 +1,27 @@
+package com.rtbhouse.grpc.loadbalancer;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/** Unit test for simple LoadBalancerServer. */
+public class LoadBalancerServerTest extends TestCase {
```

Comment:
?

Reply:
Na szczęście mamy już normalny test na LoadBalancerServer, a to nawet nie wiem jakim cudem się tu znalazło :)